README.md

```diff
@@ -666,5 +741,10 @@ Payments.modifyRailPayment(
 ```
 
 #### Best Practice
+
 - Unless absolutely required, clients should refrain from cutting operator allowances for ongoing transactions.
 - In the event that the rate stream cannot be initiated, operators should be prepared for this possibility and utilize one-time payments as a backup.
+
+## License
+
+Licensed under [MIT](https://github.com/filecoin-project/lotus/blob/master/LICENSE-MIT)
```

Comment:
```suggestion
Licensed under [MIT](https://github.com/FilOzone/filecoin-services-payments/blob/main/LICENSE)
```

Reply:
Another question is if we should add a dual-licence similar to the PDP-contract: https://github.com/FilOzone/pdp?tab=License-1-ov-file#readme

Reply:
Okay, you beat me to it: https://github.com/FilOzone/filecoin-services-payments/pull/183